src/util/lib/cosi/registration/graphs/index.js

```diff
@@ -132,11 +132,7 @@ class Graphs extends Registration {
 
                 console.log(`\tFound ${templateType}-${templateId} ${template.file}`);
 
-                if ({}.hasOwnProperty.call(self.metrics, templateId)) {
-                    self.templates.push(templateId);
-                } else {
-                    console.log(`\t${chalk.yellow('Skipping')} ${templateType}-${templateId}, no metrics found for '${templateId}'.`);
-                }
+                self.templates.push(templateId);
```

Comment:
OK this needs fixing. It was added to address (A) form below.

The exiting if-clause checks if we have a plugin/"metric group" that is named after the `templateId`, this is different from checking if there are metrics for the template. There might be very well, just not in that metric group.

I might be able to work around this by putting everyhing into the existing templates which have appropriate templateIds set...